extensions/parquet/table/src/test/java/io/deephaven/parquet/table/ParquetTableReadWriteTest.java

```diff
@@ -856,38 +853,40 @@ public void dictionaryEncodingTest() {
     @Test
     public void overflowingStringsTest() {
         // Test the behavior of writing parquet files if entries exceed the page size limit
+        final int pageSize = 2 << 10;
```

Comment:
I might have said to explicitly reference the constant, but this is fine.

Reply:
Will make the change in another PR